src/format.rs

```diff
@@ -207,13 +207,23 @@ impl<'a> Formatter<'a> {
                     nil()
                 } else {
                     ".{".to_doc()
-                        .append(flex_break(
-                            unqualified
-                                .iter()
-                                .map(|e| e.clone().to_doc())
-                                .intersperse(", ".to_doc())
-                                .collect(),
-                        ))
+                        .append(
+                            concat(
+                                unqualified
+                                    .iter()
+                                    .map(|e| {
+                                        break_(&e.clone().name, &e.clone().name).append(
+                                            if let Some(name) = &e.as_name {
+                                                format!(" as {}", name).to_doc()
+                                            } else {
```

Comment:
I think this will render a break between the name of the field being imported and the `as` as `break_` can insert a newline after it.  Could we not have the previous logic and intersperse a `delim(",")`?

Reply:
I refactored the code a bit to make it more readable.  Import as is not a `Break Document` so new line will not be inserted.  Please let me know what you think.  Will address any concern/feedback you have.  Thanks.

Reply:
Thanks :)